tests/storage/concurrent_vm_boot/utils.py

```
@@ -0,0 +1,90 @@
+    try:
+        for disk_index in range(NUM_BLANK_DISKS_PER_VM):
+            add_dv_to_vm(
+                vm=vm,
+                template_dv=blank_dv_template(
+                    name=f"{vm_name}-blank-{disk_index}",
+                    namespace=namespace_name,
+                    storage_class_name=storage_class_name,
+                ),
+            )
+    except Exception as exc:
+        LOGGER.error(f"Failed to attach disk {disk_index} to VM {vm_name}, cleaning up")
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)